src/Uno.UI.Composition/Composition/CompositionEffectBrush.skia.cs

```diff
@@ -0,0 +1,1726 @@
+									{
+										return null;
+									}
+
+									_isCurrentInputBackdrop = false;
+
+									return SKImageFilter.CreateColorFilter(
+										SKColorFilter.CreateColorMatrix(
+											new float[] // Invert Matrix
+											{
+												-1, 0,  0,  0, 1,
+												0,  -1, 0,  0, 1,
+												0,  0,  -1, 0, 1,
+												0,  0,  0,  1, 0,
+											}),
+										sourceFilter, new(bounds));
+								}
+
+								return null;
+							}
+						case EffectType.HueRotationEffect:
+							{
+								if (effectInterop.GetSourceCount() == 1 && effectInterop.GetPropertyCount() == 1 && effectInterop.GetSource(0) is IGraphicsEffectSource source)
+								{
+									SKImageFilter? sourceFilter = GenerateEffectFilter(source, bounds);
+									if (sourceFilter is null && !_isCurrentInputBackdrop)
+									{
+										return null;
+									}
+
+									_isCurrentInputBackdrop = false;
+
+									effectInterop.GetNamedPropertyMapping("Angle", out uint angleProp, out GraphicsEffectPropertyMapping angleMapping);
+									float angle = (float)(effectInterop.GetProperty(angleProp) ?? throw new NullReferenceException("The effect property was null"));
+
+									if (angleMapping == GraphicsEffectPropertyMapping.RadiansToDegrees)
+									{
+										angle *= 180.0f / MathF.PI;
+									}
+
+									return SKImageFilter.CreateColorFilter(
+										SKColorFilter.CreateColorMatrix(
+											new float[] // Hue Rotation Matrix
+											{
+												0.2127f + MathF.Cos(angle) * 0.7873f - MathF.Sin(angle) * 0.2127f, 0.715f - MathF.Cos(angle) * 0.715f - MathF.Sin(angle) * 0.715f, 0.072f - MathF.Cos(angle) * 0.072f + MathF.Sin(angle) * 0.928f, 0, 0,
+												0.2127f - MathF.Cos(angle) * 0.213f + MathF.Sin(angle) * 0.143f,   0.715f + MathF.Cos(angle) * 0.285f + MathF.Sin(angle) * 0.140f, 0.072f - MathF.Cos(angle) * 0.072f - MathF.Sin(angle) * 0.283f, 0, 0,
+												0.2127f - MathF.Cos(angle) * 0.213f - MathF.Sin(angle) * 0.787f,   0.715f - MathF.Cos(angle) * 0.715f + MathF.Sin(angle) * 0.715f, 0.072f + MathF.Cos(angle) * 0.928f + MathF.Sin(angle) * 0.072f, 0, 0,
+												0,                                                                   0,                                                                0,                                                                1, 0
+											}),
+										sourceFilter, new(bounds));
+								}
+
+								return null;
+							}
+						case EffectType.TintEffect:
+							{
+								if (effectInterop.GetSourceCount() == 1 && effectInterop.GetPropertyCount() >= 1 /* only the Color property is required */ && effectInterop.GetSource(0) is IGraphicsEffectSource source)
+								{
+									SKImageFilter? sourceFilter = GenerateEffectFilter(source, bounds);
+									if (sourceFilter is null && !_isCurrentInputBackdrop)
+									{
+										return null;
+									}
+
+									_isCurrentInputBackdrop = false;
+
+									if (Compositor.IsSoftwareRenderer is true)
```

Comment:
no, it's because SkiaSharp 2 doesn't implement SkSL on software rendering, SkiaSharp 3 fixes this so we should remove the check once we upgrade to SkiaSharp 3